applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h

```diff
@@ -36,21 +36,17 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
     explicit TransientThermalElement(IndexType NewId = 0) : Element(NewId) {}
 
     TransientThermalElement(IndexType NewId, GeometryType::Pointer pGeometry)
-        : Element(NewId, pGeometry)
+        : Element(NewId, pGeometry),
+          mIntegrationCoefficientsCalculator{IntegrationCoefficientsCalculator(
+              std::make_unique<IntegrationCoefficientModifierForThermalElement>())}
```

Comment:
```suggestion
          mIntegrationCoefficientsCalculator{std::make_unique<IntegrationCoefficientModifierForThermalElement>()}
```

This suggestion also applies to line 47.

Reply:
done